client/src/pages/pot-instance/components/AddIngredients/AddIngredients.tsx

```diff
@@ -17,7 +26,7 @@ const AddIngredients = () => {
         open={drawerOpen}
         onClose={() => toggleDrawer(false)}
       >
-        <IngredientsList />
+        <IngredientsList addFoodTimer={addFoodTimer} drawerOpen={drawerOpen} />
```

Comment:
`addFoodTimer `is the function that displays the ingredients from cart onto the state that is used to display each timer

`drawerOpen` state is added to force push contents of the cart, bypassing the debounce function